pinot-spi/src/main/java/org/apache/pinot/spi/config/table/UpsertConfig.java

```diff
@@ -60,6 +60,8 @@ public enum Strategy {
   @JsonPropertyDescription("Whether to use snapshot for fast upsert metadata recovery")
   private boolean _enableSnapshot;
 
+  private double _upsertTTLInComparisonTimeUnit;
```

Comment:
This does not necessary to be time though. How about naming it `_metadataTTL` and also add some descriptions